packages/core/test/predictableExec.test.ts

```diff
@@ -571,21 +559,96 @@ describe('predictableExec', () => {
     expect(actual).toEqual([0, 1, 2]);
   });
 
-  it('`.nextState()` should not execute actions `predictableActionArguments`', () => {
-    let spy = jest.fn();
+  it('parent should be able to read the updated state of a child when receiving an event from it', (done) => {
```

Comment:
I know this is a little bit pedantic... but those tests were not yet part of the next branch. They should only be brought over here when merging main to next.

Reply:
They were important to bring over as the work I was doing in this branch overlapped with that work.